CHANGELOG.md

```diff
@@ -23,8 +23,9 @@ and this project adheres to
 
 ## Fixed
 
-- 🐛(frontend) invalidate queries after removing user #336
-- 🐛(backend) Fix dysfunctional permissions on document create #329
+🐛(frontend) invalidate queries after removing user #336
+🐛(backend) fix nginx docker container #340
+🐛(backend) Fix dysfunctional permissions on document create #329
```

Comment:
The hyphen are missing.

Reply:
Should all be set now

Reply:
@AntoLC We should be good to go now, we now only have one commit, hyphens have been re-added.